### DISCUSSION TITLE
Is it possible to include a footnote file globally?

### DISCUSSION THREAD

**Discussion by ChristianWatkins** [quarto-dev/quarto-cli#4379] [Q&A]
Hi, I've made a global footnote file that I include in my posts on a post by post basis like this:
`{{< include ../../_footnotes.qmd >}}`
Is there a way to set this up in `/posts/_metadata.yml` or something similiar? 
I want to just reference the file once i one file instead of in each and every post.

**Comment by cderv** [comment]
Not a direct answer but usually to add some content in all docs, you can use includes (https://quarto.org/docs/reference/formats/html.html#includes). 

The after body one is usually well suited for this kind of things. 

However, this needs to be HTML and not `qmd` file. But it can be customized once per project for example
